core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -1995,7 +1948,7 @@ private void addTextImportRequest(CopyModel model, @Nullable CharSequence fileNa
                     throw SqlException.$(0, "Unable to process the import request. Another import request may be in progress.");
                 }
             } else {
-                throw SqlException.$(0, "Another import request is in progress.");
+                throw SqlException.$(0, "Another import request is in progress. Current table name = " + tableName);
```

Comment:
nit: `Current table name` part of the message seems misleading as the user might think that we mean the currently imported table, while the actual value is the attempted table's name.